src/modules/configuration/index.ts

```diff
@@ -327,6 +327,24 @@ export const configValues = () => ({
           firstDepositId: 1,
         },
       ],
+      [ChainIds.blast]: [
+        {
+          address: "0x2D509190Ed0172ba588407D4c2df918F955Cc6E1",
+          startBlockNumber: 5574280,
+          abi: JSON.stringify(SpokePoolV3Abi),
+          acrossVersion: AcrossContractsVersion.V3,
+          firstDepositId: 1,
```

Comment:
The first deposit ID for these pools is 1,000,000

Reply:
fixed!